autoendpoint/src/routes/webpush.rs

```diff
@@ -25,7 +26,9 @@ pub async fn webpush_route(
         RouterType::from_str(&notification.subscription.user.router_type)
             .map_err(|_| ApiErrorKind::InvalidRouterType)?,
     );
-
+    if notification.has_topic() {
+        app_state.metrics.incr("ua.notification.topic")?;
+    }
```

Comment:
We don't want this one, with the `ua.notification` prefix it was previously emitted by autoconnect. We consolidated it into `ua.notification.sent` in https://github.com/mozilla-services/autopush-rs/pull/330 (so I just struck it from the spreadsheet)

```suggestion
```